src/cpp/core/system/file_monitor/Win32FileMonitor.cpp

```diff
@@ -91,7 +93,8 @@ void safeCloseHandle(HANDLE hObject, const ErrorLocation& location)
    {
       if (!::CloseHandle(hObject))
       {
-         LOG_ERROR(LAST_SYSTEM_ERROR());
+         auto error = LAST_SYSTEM_ERROR();
+         core::log::logError(error, location);
```

Comment:
Why the change from `LOG_ERROR` to direct logging call? I generally support reducing pre-proc macro usage as a Good Thing, but is there any other reason here?

Reply:
The main reason was that the old version didn't use the `location` passed in from the caller.